### PR TITLE
BETA: Register sashank.is-a.dev

### PR DESCRIPTION
**Pull request by SashankBhamidi**
Added `sashank.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/sashank.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "shankypedia",
+        "email": "sashankbhamidi@gmail.com"
+    },
+    "record": {
+        "URL": "https://sashank.wiki"
```
Comment (CuteDog5695) [inline domains/sashank.json:7]: Site can't be reached

**Review by iostpa — DISMISSED** [review]

**Review by Omxpro — CHANGES_REQUESTED** [review]
cannot access your website

**Comment by shankypedia** [comment]
The website was never down, it's still online, please check with this hyperlink: https://sashank.wiki/

**Comment by Omxpro** [comment]
> The website was never down, it's still online, please check with this hyperlink: https://sashank.wiki/

great

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**